### PR TITLE
[POC] Run phoenix tests on internet explorer

### PR DESCRIPTION
**Pull request by dpakach**
## Description

add drone and nightwatch configuration for running acceptance tests in internet explorer using saucelabs.

## Related Issue

- Related issue #2008

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUILogin</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/5239/

<p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong></strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/5321/

<p>






</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUITrashbin</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/5326/

<p>







</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong></strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/5326/

<p>














</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUISharingInternalGroups</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/5334/

<p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIFavorites</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/5334/

<p>







</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUISharingInternalUsers</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/5334/

<p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong></strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/5334/

<p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong></strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/5334/

<p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUITrashbin</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/5346/

<p>









</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUITrashbin</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/5349/

<p>











</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUILogin</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/5355/

<p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong></strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/5368/

<p>





</p></details>

**Comment by PVince81** [comment]
Regarding the issue with "finally", it seems to only happen when I have the markdown and other apps enabled. The files app works fine.

I noticed that we are not loading the "core-js" lib for those apps, maybe we need to add it there.
See here how it's loaded for the files app: https://github.com/owncloud/phoenix/blob/master/apps/files/src/default.js#L1

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUISharingInternalGroups</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/5419/

<p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUISharingInternalUsers</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/5419/

<p>





</p></details>

**Comment by LukasHirt** [comment]
@dpakach @individual-it I tried pushing into this branch something that could maybe help with the finally promise error. Let me know if it helped you. If not feel free to simply drop the commit.

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong></strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/5427/

<p>




</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIFiles</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/5429/

<p>













































</p></details>

**Comment by individual-it** [comment]
current status of the tests
```
29 scenarios (8 failed, 21 passed)
322 steps (8 failed, 51 skipped, 263 passed)
41m26.477s
```

skipped renaming and upload, fixed sharing, lets see whats next

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIFiles</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/5441/

<p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIFiles</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/5442/

<p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIFiles</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/5443/

<p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIFavorites</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/5453/

<p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIFiles</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/5453/

<p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIFiles</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/5455/

<p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIFiles</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/5456/

<p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIFiles</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/5457/

<p>













































</p></details>

**Comment by individual-it** [comment]
made rename and create folder input work

new status
```
32 scenarios (6 failed, 26 passed)
329 steps (6 failed, 48 skipped, 275 passed)
51m46.475s

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIFiles</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/5465/

<p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIFiles</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/5466/

<p>





</p></details>

**Comment by individual-it** [comment]
wow, that thing can really pass
```
32 scenarios (32 passed)
329 steps (329 passed)
70m47.679s
Done in 4256.80s.
```

now need to find out, why that single test fails on Chrome

**Comment by individual-it** [comment]
the Chrome tests fail because of #2150
now waiting for #2151 to be merged

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIFiles</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/5500/

<p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong></strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/5500/

<p>







</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong></strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/5500/

<p>







</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong></strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/5509/

<p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong></strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/5511/

<p>





</p></details>

File: tests/drone/config.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
 		"authUrl": "http://owncloud/index.php/apps/oauth2/authorize"
 	},
 	"apps" : [
-		"files", "markdown-editor", "pdf-viewer", "media-viewer", "draw-io"
```
Comment (DeepDiver1975) [inline tests/drone/config.json]: this will have impact on the build command ... I'd love to keep this list.

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>iPhone</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/5593/

<p>































</p></details>

File: tests/drone/config.json (hunk 1 above)
Comment (individual-it) [reply tests/drone/config.json]: even if the other apps are not tested (yet)?

**Comment by individual-it** [comment]
to make sure IE tests still pass I've opened a temporary PR, running only them: #2208

**Pull request merged**